docs/source/crawler/basics.rst

```diff
@@ -20,12 +20,56 @@ a modified dictionary as its output once.
 >>> next(add1_node({'input': 1}))
 {'input': 2}
 
+.. Generators::
+    Generators may have been created from various different code, including,
+    but not limited to, nodes or independent functions as long as they yield.
+    Generators may yield a dictionary, an error message, or any number of Python
```

Comment:
"various different" is tautology seems to me.

overall it is still a bit incorrect ;)  e.g. generator doesn't return anything really... it can generate, when is asked to by 'next'...  as for functions/methods defining generators (i.e. having `yield` statement -- they can yield (which would result in a generator), or return None (i.e. without any specific value), so not quite like a regular function.

If you are to describe Generators here -- just find a good reference, preferably within python documentation itself, and just point there as for description of details.  The fact which is relevant here is that a "node" must be a generator which gets a dictionary and yields a dictionary 0, 1, or more times.  Operate below with a term 'node', not a generator when describing a pipeline.